Guided Investigation - Incident Triage.ipynb

```diff
@@ -57,40 +57,28 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        "matplotlib.pyplot,,plt",
</pre><p>Several of these are imported by init_notebook.</p><p>Also, the MP imports are using old paths (maybe that's OK for compatibility with pre MP 2.0 though)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>